### PR TITLE
Rtp folder reorg + bad file scanning and renaming

### PR DESCRIPTION
**Pull request by jsdillon**
This PR is trying to resolve some technical debt, where two very different ideas got put into the same branch in the chaos of the first half of H4C.

The first has to do with scanning files for bad metadata, removing them from the makeflow, and optionally renaming them something so that processes that look for neighboring files don't find them. 

The second, which I'm not sure if we still want, has to do with making multiple folders for running multiple RTPs on different data taking runs of the same day. @plaplant do we still want that, or should we strip it out of this PR?

### CODE REVIEW

File: scripts/make_rtp_workflow.py [hunk 1]
```diff
@@ -105,20 +110,29 @@
                 continue
 
         # make target directory if it doesn't exist
-        date = datetime.date.today().strftime("%y%m%d")
-        MF_LOCATION = os.path.join("/home/obs/rtp_makeflow", date)
-        if not os.path.isdir(MF_LOCATION):
-            os.makedirs(MF_LOCATION)
+        jd0 = float(new_files[0][4:17])
```
Comment (jsdillon) [inline scripts/make_rtp_workflow.py:113]: I'm not sure if we still want to be doing this...

File: scripts/make_rtp_workflow.py [hunk 2]
```diff
@@ -133,7 +147,13 @@
             subprocess.check_call(screen_cmd1)
             subprocess.check_call(screen_cmd2)
         except subprocess.CalledProcessError as e:
-            raise ValueError(
+            # raise ValueError(
```
Comment (jsdillon) [inline scripts/make_rtp_workflow.py:150]: Not sure why this was commented out...

**Comment by plaplant** [comment]
@jsdillon thanks for the PR. I'm happy to add the file scanning capability to the `build_makeflow_from_config.py` script, as it seems useful. I don't think we want to make any meaningful changes to the `make_rtp_workflow.py` script, because going forward it won't be used anymore. (It used to be the way we launched RTP, but that is going to be replaced by hera_mc.) I think we should take those changes out of this PR, but I'm happy to talk about including lessons learned from it into the M&C functionality.

**Review by plaplant — COMMENTED** [review]
The changes look great, thanks! I'll take the changes for `make_rtp_workflow.py` out of this PR and then I think this is good to go.

**Review by plaplant — APPROVED** [review]

**Pull request merged**